src/Apps/Settings/SettingsApp.tsx

```diff
@@ -44,48 +41,20 @@ const SettingsApp: React.FC<SettingsAppProps> = ({ me, children }) => {
     })
   }, [location.query.error, sendToast])
 
-  const tabsWhenCollectorProfileEnabled = compact([
-    { name: "Edit Profile", url: "/settings/edit-profile" },
-    { name: "Saved Alerts", url: "/settings/alerts" },
-    { name: "Account", url: "/settings/edit-settings" },
-    { name: "Order History", url: "/settings/purchases" },
-    { name: "Bids", url: "/settings/auctions" },
-    { name: "Payments", url: "/settings/payments" },
-    { name: "Shipping", url: "/settings/shipping" },
-  ])
-
-  if (!isLoggedIn) {
-    return <MyCollectionRouteLoggedOutState />
-  }
-
   return (
     <>
       <MetaTags title="Settings | Artsy" />
 
-      <>
-        <Breadcrumbs py={[2, 1]}>
```

Comment:
This was a bit of abuse of our design system: These aren't breadcrumbs, which leave a trail of the hierarchal relationship:

![](https://static.damonzucconi.com/_capture/vcq0EF5P.png)

We also would never want to just overwrite all the styles to it.

What you seemed to be looking for is the `TopContextBar` which describes the current context under which a page might be nested. Like a singular breadcrumb. It's much easier to use and does not require messing with the styling.